contrib/dist/publish_doc

```diff
@@ -36,7 +36,7 @@ if ! test -f VERSION; then
   exit 1
 fi
 
-VERSION=$(./config/hwloc_get_version.sh ./VERSION)
+VERSION=$(sh /config/hwloc_get_version.sh ./VERSION)
```

Comment:
I think you need to leave the `.` here, or ditch the `/`

Reply:
Sorry, I force-pushed it just before seeing your comments. I hope it's good now!